From 2bff60433f9dd320d864ae68431bfc19c1f7194e Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Mon, 7 Sep 2020 14:58:54 +0800
Subject: [PATCH] arm64: dts: rockchip: Assign GMAC_ACLK to 180M for rk3328

If aclk_gmac is not set, the default configuration
is 98.304M, which cause the tcp checksum error.

Signed-off-by: David Wu <david.wu@rock-chips.com>
Change-Id: I7009541f6a035285d038f84d7a4222aba26beed0
---
 arch/arm64/boot/dts/rockchip/rk3328.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index ea52f1a18e12..c3952f74a26b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -1116,7 +1116,8 @@
 			<&cru ACLK_BUS_PRE>, <&cru HCLK_BUS_PRE>,
 			<&cru PCLK_BUS_PRE>, <&cru ACLK_PERI_PRE>,
 			<&cru HCLK_PERI>, <&cru PCLK_PERI>,
-			<&cru SCLK_RTC32K>, <&cru ACLK_VOP>;
+			<&cru SCLK_RTC32K>, <&cru ACLK_VOP>,
+			<&cru ACLK_GMAC>;
 		assigned-clock-parents =
 			<&cru HDMIPHY>, <&cru PLL_APLL>,
 			<&cru PLL_GPLL>, <&xin24m>,
@@ -1136,7 +1137,8 @@
 			<150000000>, <75000000>,
 			<75000000>, <150000000>,
 			<75000000>, <75000000>,
-			<32768>, <400000000>;
+			<32768>, <400000000>,
+			<180000000>;
 	};
 
 	usb2phy_grf: syscon@ff450000 {
-- 
2.35.3

